File: 02_processing_data/subset_variables/subset_variables.py
# - subset variables ----
from sklearn.datasets import load_iris
import pandas as pd

data = load_iris()
df = pd.DataFrame(data.data, columns=data.feature_names)
df.head()
data = df[['sepal length (cm)']] # conditionally select the variables to remove from the dataframe. 
print(data.head())

File: 02_processing_data/subset_variables/subset_variables.r
# - subset variables ----
library(datasets) # datasets library for sample data...
data(iris) # import into the r enviroment... 
data = iris # store the iris dataframe as 'data' in the r enviroment...
remove = c("Petal.Width")
data = data[, !colnames(data) %in% c(remove)] # conditionally select the variables to remove from the dataframe. 
print(str(data))

subset_var <- function(x,y){
  remove = c(toString(y))
  sub = x[, !colnames(x) %in% c(remove)] # conditionally select the variables to remove from the dataframe. 
  print(str(sub))
}

subset_var(data,'Petal.Width')
